truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -277,6 +278,7 @@ protected Debugger createDebugger(Object vm, Instrumenter instrumenter) {
     @SuppressWarnings("unused")
     protected Closeable executionStart(Object vm, int currentDepth, Debugger debugger, Source s) {
         vm.getClass();
```

Comment:
Feel free to merge this PR anytime you want. I think I am done with it.

----- Reply message -----
Od: "Andreas Woess" <notifications@github.com>
Komu: "graalvm/truffle" <truffle@noreply.github.com>
Kopie: "Jaroslav Tulach" <jaroslav.tulach@oracle.com>
Předmět: [truffle] Making sure invokeForeign as wellas executionStart are never include… (#36)
Datum: pá, úno 5, 2016 19:42

In truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java:
> @@ -277,6 +278,7 @@ protected Debugger createDebugger(Object vm, Instrumenter instrumenter) {
>      @SuppressWarnings("unused")
>      protected Closeable executionStart(Object vm, int currentDepth, Debugger debugger, Source s) {
>          vm.getClass();

As you wish. I'll create a PR once this is merged (to avoid conflicts).

&mdash;
Reply to this email directly or view it on GitHub.